src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -233,6 +238,15 @@ public void processElement(ProcessContext context) {
                             if (!event.isEmpty()) {
                               builder.withEvent(event);
                             }
+                            
+                            String fields = metadata.optString(HEC_FIELDS_KEY);
+                            if (!fields.isEmpty()) {
+                              try {
+                                builder.withFields(GSON.fromJson(fields, JsonObject.class));
+                              } catch (JsonParseException e) {
+                                LOG.warn("Unable to parse 'fields' metadata JSON");
```

Comment:
Can you include fields content in the message as 
```LOG.warn("Unable to convert 'fields' metadata value:{} into JSON object ", fields)```

Reply:
Done